content/pages/health-care/index.md

```diff
@@ -16,6 +21,9 @@ majorlinks:
     - url: /health-care/apply/
       title: Application Process
       description: Apply online now, or find out how to apply in person, by phone, or by mail.
+      sibling: true
```

Comment:
I think on the pension page we put the status above the major links, rather than in them.

Reply:
Thanks @jbalboni @peggygannon for pointing this out. I had made this change based on @stacyhlinn [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5268#issuecomment-336021727) on the original issue. Thanks all for confirming which way the card should be displayed.